CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 #### Enhancements
 
+* Add `LowerACLThanParent` rule
```

Comment:
Hi @keith, please leave [two trailing spaces](https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes) after the description.

Reply:
Done!